app/src/client/utils/chatUtils.ts

```diff
@@ -8,7 +8,7 @@ import {
 } from 'wasp/client/operations';
 
 import { type Conversation, type Chat } from 'wasp/entities';
-import { SelectedModelSchema } from '../interfaces/BuildPageInterfaces';
+import { SelectedModelSchema } from '../interfaces/BuildPageInterfacesNew';
```

Comment:
Like previous comment if old BuildPageInterfaces no longer exists maybe we can keep the same name.

Reply:
I have updated the filename from `BuildPageInterfacesNew ` to `BuildPageInterfaces `.